Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -1845,6 +1848,25 @@ WI._updateModifierKeys = function(event)
         WI.notifications.dispatchEventToListeners(WI.Notification.GlobalModifierKeysDidChange, event);
 };
 
+WI._updateAccentColorClass = function()
+{
+    let testElement = document.createElement("div");
+    testElement.style.color = "-apple-system-control-accent";
+    testElement.style.position = "absolute";
+    testElement.style.visibility = "hidden";
+    document.body.appendChild(testElement);
+    let accentColorString = getComputedStyle(testElement).color;
+    document.body.removeChild(testElement);
+
+    let accentColor = WI.Color.fromString(accentColorString);
+    if (!accentColor)
+        return;
+
+    let hsl = accentColor.hsl;
+    if (hsl[1] < 15)
```

Comment:
Is this value sufficient to cover the case when Graphite is the system accent color?